client/components/App.jsx

```diff
@@ -63,16 +63,14 @@ class App extends React.Component {
       return (null);
```

Comment:
An interesting alternative to having an initial state.  How does it look in action?

Reply:
I really like it. Everything I've done up til now I implemented very differently. It's been a bit of a learning curve. Was encouraged to try it by Matt and a SEIR.